D1673/P1673.md

```diff
@@ -4126,7 +4160,7 @@ concept @_out-object_@; = // exposition only
 template<class T>
 concept @_inout-object_@; = // exposition only
 
-// [linalg.algs], algorithms
+// [linalg.algs.blas1], BLAS-1 Functions
```

Comment:
I do suppose they are "functions," but should we consider calling them "BLAS 1 algorithms," in order to emphasize how they are related to the existing Standard Algorithms?

```suggestion
// [linalg.algs.blas1], BLAS 1 algorithms
```

Reply:
Yeah sounds good